resources/assets/js/components/Fields/JsTree.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <div class="form-group">
+        <div id="js-tree"></div>
+    </div>
+</template>
+
+<script>
+    import {serverBus} from '../../laradium';
+
+    export default {
+        props: ['tree', 'field'],
+
+        data () {
+            return {
+                has_new_items: false,
+                new_item_count: 0
+            };
+        },
+
+        created() {
+            let $vm = this;
+
+            serverBus.$on('added_tree_item', function (id) {
```

Comment:
```suggestion
            serverBus.$on('added_tree_item', id => {
```

Same as before - use arrow functions